ibis/expr/operations/generic.py

```diff
@@ -195,6 +195,9 @@ def name(self):
         return repr(self.value)
 
 
+NULL = Literal(None, dt.null)
```

Comment:
This is the definition of `ibis.NA`:

```
❯ rg 'def null\(' -A 10
ibis/expr/types/generic.py
1988:def null():
1989-    """Create a NULL/NA scalar."""
1990-    return literal(None)
```

Can we define `ibis.NA` as:

```
NA = ops.NULL  # or ops.generic.NULL?
```

to avoid duplicating the definition?

Reply:
Yes, and we can also turn it into a singleton

Reply:
Didn't turn it into a singleton, because there isn't just a single NULL. We interpret `None` as `NULL` nonetheless the datatype.